Library/Homebrew/cmd/install.rb

```diff
@@ -396,6 +396,20 @@ def install_formula(f, args:)
     f.print_tap_action
     build_options = f.build
 
+    if f.outdated? && !f.head?
+      formulae = [f] + f.old_installed_formulae
```

Comment:
```suggestion
      formulae = [f, *f.old_installed_formulae]
```

Reply:
What's the reason for this change? Just curious, they seem functionally equivalent to me.

Reply:
The latter avoids creating a two new arrays/mutating any in favour of just creating one.

Reply:
Makes sense, thanks for the explanation!